test/Interop/Cxx/class/linked-records-module-interface.swift

```diff
@@ -24,20 +24,14 @@
 // CHECK:   init()
 // CHECK: }
 // CHECK: struct F {
-// CHECK:   struct __Unnamed_union___Anonymous_field0 {
-// CHECK:     struct __Unnamed_struct_c {
-// CHECK:       init()
-// CHECK:     }
-// CHECK:     var c: F.__Unnamed_union___Anonymous_field0.__Unnamed_struct_c
-// CHECK:     var m: M
-// CHECK:     init()
-// CHECK:     init(c: F.__Unnamed_union___Anonymous_field0.__Unnamed_struct_c)
-// CHECK:     init(m: M)
-// CHECK:   }
-// CHECK:   var __Anonymous_field0: F.__Unnamed_union___Anonymous_field0
```

Comment:
This test is incorrect. I'm going to reset this change before I land the patch (and I'll make sure the CI passes).